app/controllers/katello/api/rhsm/candlepin_proxies_controller.rb

```diff
@@ -204,10 +204,12 @@ def enabled_repos
 
     #api :POST, "/environments/:environment_id/consumers", N_("Register a consumer in environment")
     def consumer_create
+      fail "Unable to register system, not all services available" unless Katello::RegistrationManager.check_registration_services
```

Comment:
I would move this out to a before_filter, also missing i18n `_("unable to register system)`